src/DefaultBuilder/src/BootstrapHostBuilder.cs

```diff
@@ -157,7 +162,9 @@ private class HostingEnvironment : IHostEnvironment
             public string EnvironmentName { get; set; } = default!;
             public string ApplicationName { get; set; } = default!;
             public string ContentRootPath { get; set; } = default!;
+            public string WebRootPath { get; set; } = default!;
             public IFileProvider ContentRootFileProvider { get; set; } = default!;
+            public IFileProvider WebRootFileProvider { get; set; } = default!;
```

Comment:
I don't think it's necessary to add these properties or update BootstrapHostBuilder at all for this change because this doesn't need to be an IWebHostEnvironment. We get that [here](https://github.com/dotnet/aspnetcore/blob/711bdcf96afbbbe122a302ddc641accea0768431/src/DefaultBuilder/src/WebApplicationBuilder.cs#L95) after it's been [created by the GenericWebHostBuilder](https://github.com/dotnet/aspnetcore/blob/711bdcf96afbbbe122a302ddc641accea0768431/src/Hosting/Hosting/src/GenericHost/GenericWebHostBuilder.cs#L402) which should already read the config and create the right FileProvider.

Reply:
Correct. The fact that it is set in the configuration suffices for our purposes. TY!